airbyte-server/src/main/resources/application.yml

```diff
@@ -12,6 +12,10 @@ micronaut:
         web:
           allowedOrigins:
             - ^.*$
+    netty:
+      access-logger:
+        enabled: ${HTTP_ACCESS_LOG_ENABLED:true}
```

Comment:
what does this do?

Reply:
Enables request logging.  You can see an example of what the output looks like by default in the description of this PR:

```
2023-02-17 16:58:01 INFO i.m.h.s.n.h.a.e.AccessLog(log):125 - airbyte-webapp.airbyte_airbyte_internal - - [17/Feb/2023:16:58:01 +0000] "POST /api/v1/workspaces/get HTTP/1.0" 200 298
2023-02-17 16:58:07 INFO i.m.h.s.n.h.a.e.AccessLog(log):125 - airbyte-webapp.airbyte_airbyte_internal - - [17/Feb/2023:16:58:07 +0000] "POST /api/v1/workspaces/update HTTP/1.0" 200 390
2023-02-17 16:58:07 INFO i.m.h.s.n.h.a.e.AccessLog(log):125 - airbyte-webapp.airbyte_airbyte_internal - - [17/Feb/2023:16:58:07 +0000] "POST /api/v1/web_backend/connections/list HTTP/1.0" 200 18
2023-02-17 16:58:07 INFO i.m.h.s.n.h.a.e.AccessLog(log):125 - airbyte-webapp.airbyte_airbyte_internal - - [17/Feb/2023:16:58:07 +0000] "POST /api/v1/destination_definitions/list_for_workspace HTTP/1.0" 200 119635
2023-02-17 16:58:07 INFO i.m.h.s.n.h.a.e.AccessLog(log):125 - airbyte-webapp.airbyte_airbyte_internal - - [17/Feb/2023:16:58:07 +0000] "POST /api/v1/source_definitions/list_for_workspace HTTP/1.0" 200 1290066
2023-02-17 16:58:08 INFO i.m.h.s.n.h.a.e.AccessLog(log):125 - airbyte-webapp.airbyte_airbyte_internal - - [17/Feb/2023:16:58:07 +0000] "POST /api/v1/web_backend/check_updates HTTP/1.0" 200 50
2023-02-17 16:58:21 INFO i.m.h.s.n.h.a.e.AccessLog(log):125 - airbyte-webapp.airbyte_airbyte_internal - - [17/Feb/2023:16:58:21 +0000] "GET /api/v1/health HTTP/1.0" 200 18
```